examples/nextjs/pages/_app.tsx

```diff
@@ -16,7 +15,7 @@ if (devTools) {
     iframe: false
   })
 }
-const nhost = new NhostClient({ backendUrl: BACKEND_URL, devTools })
+const nhost = new NhostClient({ subdomain: 'localhost', devTools })
```

Comment:
We should change every occurrence of `subdomain: 'localhost'` to `subdomain: 'local'` later